api/jwt/token_blacklist.go

```diff
@@ -0,0 +1,86 @@
+package jwt
+
+import (
+	"sync"
+	"time"
+)
+
+type blacklistToken struct {
+	token        string
+	creationTime int64
+}
+
+type BlacklistTokenMap struct {
+	blacklist map[string]*blacklistToken
+	lock      sync.Mutex
+	ttl       int64
+}
+
+// New Creates a new blacklist for invalid tokens. The timeout for the element is given as integer
+func New(maxTimeToLive int, interval time.Duration) (m *BlacklistTokenMap) {
```

Comment:
Misleading name:

`New()` is discouraged because it could be taken for the whole package. It is misleading in that way that no new JWT will be created but a blocklist token map instead.

`NewBlocklistTokenMap()` or something would remove the misleading idea.